runtime/caml/roots.h

```diff
@@ -32,9 +32,9 @@ extern uintnat caml_incremental_roots_count;
 CAMLextern void caml_do_local_roots (scanning_action, value *, value *,
                                      struct caml__roots_block *);
 #else
-CAMLextern void caml_do_local_roots(scanning_action f, char * bottom_of_stack,
-                                    uintnat last_retaddr, value * gc_regs,
-                                    struct caml__roots_block * local_roots);
```

Comment:
If the only collision is only the local root names, could we not just rename `local_roots` to `gc_local_roots`?